Using device: TITAN V
Moved to GPU
Traceback (most recent call last):
  File "train_mlp.py", line 118, in <module>
NameError: name 'batch_idx' is not defined
